spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/AbstractBinderPropertyKeysAccessor.java

```diff
@@ -29,13 +29,13 @@
  *
  * @author Gary Russell
  */
-public abstract class AbstractBinderPropertiesAccessor implements BinderProperties {
+public abstract class AbstractBinderPropertyKeysAccessor extends BinderPropertyKeys {
```

Comment:
This should *not* be renamed since it is the accessor for actual property *value*. It's not an *implementation* of the `BinderPropertyKeys` interface. In fact, I don't think we should have the *extends* keyword here, but rather should just have a dependency on `BinderPropertyKeys` where those constants are needed.